### PR TITLE
Added the kbd bundle

### PR DESCRIPTION
**Pull request by bauen1**
Its time to end this madness, load a german keyboard layout using:
```bash
loadkeys de
```

This bundle adds about 4.6mb, you could reduce its size by removing some of the generated keymaps and consolefonts in `/usr/share/`

### CODE REVIEW

**Comment by ivandavidov** [comment]
Just for your information - long time ago the user **ladiko** added German keyboard layout but back then I ignored this feature because... well, I'm using the standard en-US keyboard layout and I have no troubles at all. :)

https://github.com/ladiko/minimal/blob/master/src/5_generate_rootfs.sh

**Comment by ivandavidov** [comment]
Note to myself - it is a good idea to provide corresponding 'autorun' script which has a commented entry which changes the keyboard layout to something else. The same script should print some help message in the console which explains how to change the keyboard layout.

This will help all future MLL users to either change the keyboard layout automatically (via the autorun script), or to change the keyboard layout manually with simple command.

This should also be documented more thoroughly in the ``.config`` and ``README`` files.

**Comment by ivandavidov** [comment]
Note that I have removed the configuration line where all overlay bundles are explicitly mentioned. It doesn't make sense anymore.

File: src/.config [hunk 1]
```diff
@@ -231,7 +237,7 @@ COPY_SOURCE_ISO=true
 #
 # Refer to the README file for more information.
 #
-#OVERLAY_BUNDLES=glibc_full,cf_cli,coreutils,dhcp,dropbear,felix,java,links,lua,mll_utils,nano,ncurses,nweb,openjdk,static_get,util_linux,vim,zlib
+#OVERLAY_BUNDLES=glibc_full,cf_cli,coreutils,dhcp,dropbear,felix,java,kbd,links,lua,mll_utils,nano,ncurses,nweb,openjdk,static_get,util_linux,vim,zlib
```
Comment (bauen1) [inline src/.config]: This one ?

Comment (ivandavidov) [reply src/.config]: Oh fuck...

Comment (ivandavidov) [reply src/.config]: Yep, this one. There should have been something else as a helper message but it's not there.

Comment (ivandavidov) [reply src/.config]: I'll fix it later.

**Comment by ivandavidov** [comment]
I have implemented small additional help screen as 'autorun' script which is displayed on boot. I will push the change along with some other changes that I'm currently developing.

**Pull request merged**